molgenis-core-ui/src/main/javascript/molgenis-global-webpack.js

```diff
@@ -35,3 +35,6 @@ window.top.molgenis.isXrefAttr = isXrefAttr;
 window.top.molgenis.isMrefAttr = isMrefAttr;
 window.top.molgenis.isCompoundAttr = isCompoundAttr;
 
+import { htmlEscape } from 'utils/HtmlUtils';
+
+window.top.molgenis.htmlEscape = htmlEscape;
```

Comment:
I think that isn't quite right. The "old" code put the htmlEscape function on the global/window scope. Now it gets put in the molgenis object, but `molgenis-dataexplorer/src/main/resources/js/dataexplorer-filter.js` and `molgenis-core-ui/src/main/resources/js/jquery.molgenis.xrefmrefsearch.js ` assume it's in the global scope.